### DISCUSSION TITLE
Is there any way to save and then reuse the model?

### DISCUSSION THREAD

**Discussion by delustra** [asieradzk/RL_Matrix#5] [General]
I'm playing with the RLMatrix and thinking whether it's possible to save the state of the model and then reload it later?

**Comment by asieradzk** [comment]
Hey, thanks for trying RLMatrix :) 

Agent types have built in method for saving and loading

PPO has .SaveModel that accepts path to dir as argument

DQN has .SaveModel method that accepts path and bool to also save replay buffer

I didn't test these methods in a long time (I made a bunch of changes since) so let me know if things break.
